### PR TITLE
New NFSD 'Arrow' Fighter Ship

### PR DESCRIPTION
**Pull request by EaglePhntm**
## About the PR

Added A new nfsd ship, Arrow.

## Why / Balance

Literally made to be like a fighter plane, i wanted to make it mostly a cockpit but need of engineering got in the way etc, but 
i think its perfectly as i intended right now with size and all, I felt there could be armed units in nfsd, although it looks like svalin does not do a good job in disabling thrusters or anything, yet to be seen how useful in pursuit, it requires two man crew in the little fighter or the solo pilot to unman their seat for several seconds to rearm the guns, unless switched with better weapons in shift. It spins quite fast but space bar easily stops it by the seems of it, the normal ship thrusters on it make the fighter really quick to get up to speed.

It is intentionally coffin like, it is sort of supposed to be like a plane cockpit except a little more for engineering etc, simultaneously made to be manned by one or two people at most like a cop car or something. No sense can be made off my thoughts.

## How to test

Spawn the ship and try. It's my first time mapping on ss14 dont kill me.

## Media

[Le vid](https://youtu.be/eylE-VTK1Es)

pics are after few tweaks, vid is old design.
image
image
image
image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Added Arrow, Small NFSD Ship

### CODE REVIEW

**Comment by dvir001** [comment]
Please provide 2 images of the ship in mapping

With and without cables.

**Comment by EaglePhntm** [comment]
> Please provide 2 images of the ship in mapping
> 
> With and without cables.

done

**Comment by dvir001** [comment]
As posted before on Discord this ship can use extending

image

**Comment by EaglePhntm** [comment]
> As posted before on Discord this ship can use extending
> 
> image

think it sort of defeats the thing i was trying to do like that, you'd need like 4 people instead of the targeted 1-2 just to fire both guns plus the cockpit, and everything else just dead space for the most part, its not like the fighter plane i was thinking of...

**Comment by EaglePhntm** [comment]
bros merge my pr

**Comment by Tych0theSynth** [comment]
> bros merge my pr

A maptainer has to review your ship prior to merging, and then you'll need to perform any (requested) changes. You'll need to be patient, as there are always quite a few ship PRs floating around and the maptainers only review at weekends. Don't be surprised if you have to wait at least a couple of weeks before they get around to you! Getting a ship added is a slow process.

**Comment by MagnusCrowe** [comment]
I'll review this tonight

**Comment by MagnusCrowe** [comment]
I have had a look and there are some things about it that I do not like very much.  I am having trouble putting my issues into a concise, coherent, and constructive statement.  I am going to take some time to think it over and get back to you.

**Comment by EaglePhntm** [comment]
> I have had a look and there are some things about it that I do not like very much. I am having trouble putting my issues into a concise, coherent, and constructive statement. I am going to take some time to think it over and get back to you.
> 
> In the meantime there's some technical issues with the PR. image image As you can see the shuttle does not show properly in the terminal or on radar.

i gotta reduce the description or something? I don't know the issue for the radar.

**Comment by arimah** [comment]
> I don't know the issue for the radar.

The shuttle is appraised to $0 ("Shuttle Estimated Value: $0"), and there is no IFF on the radar.

File: Resources/Prototypes/_NF/Shipyard/Nfsd/arrow.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  group: Security
+  shuttlePath: /Maps/_NF/Shuttles/Nfsd/arrow.yml
+
+- type: gameMap
+  id: Arrow
+  mapName: 'NSF Arrow'
+  mapPath: /Maps/_NF/Shuttles/Nfsd/arrow.yml
+  minPlayers: 0
+  stations:
+    Empress:
```
Comment (arimah) [inline Resources/Prototypes/_NF/Shipyard/Nfsd/arrow.yml:20]: This should be `Arrow`

File: Resources/Prototypes/_NF/Shipyard/Nfsd/arrow.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      stationProto: StandardFrontierSecurityVessel
+      components:
+        - type: StationNameSetup
+          mapNameTemplate: 'Arrow {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: '14'
+        - type: StationJobs
+          overflowJobs: []
+          availableJobs:
```
Comment (arimah) [inline Resources/Prototypes/_NF/Shipyard/Nfsd/arrow.yml:30]: Remove `overflowJobs` and change `availableJobs:` to `availableJobs: {}`

**Comment by EaglePhntm** [comment]
sorry i didnt know what i was doing its my first map lmao

**Comment by MagnusCrowe** [comment]
I think the major problem I have with the ship basically boils down to it being a war ship. It's a ship designed with one purpose in mind, shooting shit. 

That's not really the right vibe for an organization whos mission is to protect and serve.  This is less of a police vessel and more of a military vessel.

If you want this to be merged, I believe it would be necessary to rebrand this as a pirate or syndicate ship.  The weapons are fine the way they are for either but not for an NFSD ship.

**Comment by EaglePhntm** [comment]
> I think the major problem I have with the ship basically boils down to it being a war ship. It's a ship designed with one purpose in mind, shooting shit.
> 
> That's not really the right vibe for an organization whos mission is to protect and serve. This is less of a police vessel and more of a military vessel.
> 
> If you want this to be merged, I believe it would be necessary to rebrand this as a pirate or syndicate ship. The weapons are fine the way they are for either but not for an NFSD ship.

thought it would be used in pursuits and so on in order to disable a ship, and for deputies that would stay in a station or something and be deployed when necessary since this is not a life sustainable ship, but up to you. i guess  could be syndicate ship either but the guns dont really work against engines or anything sadly so its not really useful right now in any angle. I don't know if its up to syndie standards i never played em, nor pirates.

**Comment by arimah** [comment]
Magnus requested changes on this ship three weeks ago and nothing has been done. I'm closing this PR until such changes have been made. Please ping a maintainer if you wish to reopen this PR. :)

**Pull request closed without merging**